cognee/infrastructure/databases/graph/networkx/adapter.py

```
@@ -139,49 +139,80 @@
+        neighbours = predecessors + successors
 
-        if len(neighbour_ids) == 0:
+        return neighbours
+
+    async def get_connections(self, node_id: str) -> list:
+        if not self.graph.has_node(node_id):
+            return []
+
+        node = self.graph.nodes[node_id]
+
+        if not "uuid" in node:
```

Ruff

Test for membership should be `not in`

Convert to `not in`

(E713)